README.txt

```diff
@@ -84,6 +84,10 @@ as it is needed by the container that will run James.
 - SHA1 (optional): is the given git SHA1 of the james-project repository to build or trunk if none.
 - -s option: given tests will not be played while building. Not specifying means play tests.
 
+Some tests needs a DOCKER_HOST environment variable in order to be played, they will be ignored if you don't provide this variable.
+If you wish to play them, you may use a command like the following (depending on your docker configuration):
+$ docker run --env DOCKER_HOST=tcp://172.17.0.1:2376 -v $PWD/.m2:/root/.m2 -v $PWD:/origin -v $PWD/dockerfiles/run/spring/destination:/destination -t james/project SHA1
```

Comment:
you should document the unix socket mount strategy to play the tests on a out of the box docker install